compiler/core/src/location.rs

```diff
@@ -43,7 +43,7 @@ impl Location {
 
     pub fn reset(&mut self) {
         self.row = 1;
-        self.column = 1;
+        self.column = 0;
```

Comment:
maybe this is the reason `test_global` is broken.

Reply:
That change make so many other test pass for example in `test_ast` that I don't think the `test_global` error comes from it.
There may have been 2 problems hiding/cancelling each other for `test_global` to be OK.

Reply:
I guess that's consistent with the CPython `ast` module, but it seems like a fairly significant change.